alluxio-ui/master/src/constants/data/footerNavigation.tsx

```diff
@@ -26,4 +26,8 @@ export const footerNavigationData : INavigationData[] = [{
 }, {
   innerText: 'Resources',
   url: 'https://alluxio.org/alluxio-resources/'
+}, {
+  attributes: {target: '_blank'},
```

Comment:
Slightly unrelated to this PR, but should the other links also open in a new tab?

Reply:
Oh, good catch, I'll take a look and open them in a new tab if they make sense.